operations/patch_module.go

```diff
@@ -57,14 +61,23 @@ func PatchSetModule() cli.Command {
 				return errors.Wrap(err, "problem accessing evergreen service")
 			}
 
-			existingPatch, err := ac.GetPatch(patchID)
+			var existingPatch *patch.Patch
+			if patchID == "" {
+				patchList, err := ac.GetPatches(1)
+				if err != nil || len(patchList) != 1 {
```

Comment:
in the case that the error is len(patchList) != 1 , then we'll return a nil error since we're wrapping a nil error. We also could return something more relevant in this case ("no patches for user" if len(patchList) == 0)

Reply:
+1